### PR TITLE
Integrated Mayhem

### PR DESCRIPTION
**Pull request by bmwtsn098**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM golang:1.18
```
Comment (xansec) [inline Dockerfile]: Can we use a package stage here?

File: Mayhemfile [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Namespaced project name that the target belongs to
```
Comment (xansec) [inline Mayhemfile:1]: Multiple commands are not supported; please either use only one command or split into multiple Mayhemfiles. Also, omit image name and use generic project name.

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Comment by bmwtsn098** [comment]
I made the requested changes. Let me know if everything looks good. Thanks

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**